app/services/manual_runner.rb

```diff
@@ -0,0 +1,39 @@
+class ManualRunner
+
+  def initialize(repo)
+    raise "Repo doesn't exist" unless Repo.exist?(repo)
+    @repo = repo
+  end
+
+  def run_commits_since(date)
+    fetched_commits = fetch_commits_since(date)
+    formated_commits = format_commits(fetched_commits)
+
+    CommitsRunner.run(formated_commits)
+  end
+
+  private
+
+  def fetch_commits_since(date)
+    Octokit.auto_paginate = true
```

Comment:
Yeah, this will cause 2000 commits to be fetched into memory.

Maybe I could disable `auto_paginate` and enqueue commits after each fetched page instead of fetching all and enqueuing them all after?